backends-velox/pom.xml

```diff
@@ -318,6 +268,27 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>build-helper-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>add-cpp-libraries</id>
+            <phase>generate-resources</phase>
+            <goals>
+              <goal>add-resource</goal>
+            </goals>
+            <configuration>
+              <resources>
+                <resource>
+                  <directory>${cpp.releases.dir}</directory>
+                  <targetPath>${platform}/${arch}</targetPath>
```

Comment:
Weird. It seems that Intellij IDEA doesn't pick up this line and put the libraries directly under the root resource dir. Investigating.